src/quicr_server_raw_session.cpp

```diff
@@ -323,16 +319,16 @@ ServerRawSession::handle_publish_intent(
     publish_namespaces[intent.quicr_namespace] = context;
   } else {
     auto state = publish_namespaces[intent.quicr_namespace].state;
+    // NOLINTBEGIN(bugprone-branch-clone)
     switch (state) {
       case PublishIntentContext::State::Pending:
-        // TODO: Resend response?
-        break;
+        [[fallthrough]]; // TODO(trigaux): Resend response?
```

Comment:
I assume the intent is to do something here and not actually fall through?

Reply:
The `[[fallthrough]]` just consolidates the `break` statements.  No change to behavior intended, until we resolve the `TODO`s here.